### DISCUSSION TITLE
How to Implement Contour Maps for Sea Level

### DISCUSSION THREAD

**Discussion by shaxiu** [maplibre/maplibre-gl-js#3218] [Q&A]
### Issue Description:
I am working on a project where I need to implement contour maps to visualize sea level, similar to the effect on the [WindGuru contour map](https://www.windguru.cz/map/?lat=29.58308356054451&lon=122.23685433298624&zoom=7.740294584584762) website. The contour map should represent areas with high sea levels in darker colors and areas with lower sea levels in lighter colors. However, it seems that this is not a typical heatmap concept.

### Specific Questions:

- How can I implement contour maps for sea level in my project?

### Demo screen shot
![image](https://github.com/maplibre/maplibre-gl-js/assets/57855264/6fe86b0a-25e6-461e-aa59-c44bd622bf9f)

**Comment by HarelM** [comment]
From the top of my head:
Create contour polygons.
Each polygon has a different elevation value.
Color the polygon according to this elevation value.
I'm not sure it will result in the above image, but it might prove to pass the same information.

If you have a better mock up of what you want to achieve I might be able to think of other solutions.

You can also ask in the slack channel, there are many weather app developers there I believe.

**Reply by shaxiu** [reply]
Thank you for your response, I may need to reconsider this.
